### DISCUSSION TITLE
Table's starting value = [[]] can't add row

### DISCUSSION THREAD

**Discussion by tertris-me** [Akascape/CTkTable#42] [Q&A]
I am expecting a starting value of [[]] table to add a row after I press a the "add a row" button (self.change), but it doesn't react after I press the button.

This is my code:
```
class App(customtkinter.CTk):

    def __init__(self):
        super().__init__()
        starting_value = [[]]
        self.info = CTkTable.CTkTable(self, values=starting_value)
        self.info.grid(row=0, column=0)

        self.change = customtkinter.CTkButton(self, text="add a row", command=self.change_callback)
        self.change.grid(row=1, column=0)

    def change_callback(self):
        self.info.add_row([["a"]])


root = App()
root.mainloop()
```

This is my interface before and after I click the button:
![image](https://github.com/Akascape/CTkTable/assets/127453231/a6544e6a-a98c-4135-9783-8dbae7441dd5)

But when I make the `starting_value = [["starting","value"]]`, everything works as intended.

Interface before I pressed the button:
![starting value before](https://github.com/Akascape/CTkTable/assets/127453231/3c0de49c-96e5-4fd8-8284-edf81584929c)

Interface after:
![starting value after](https://github.com/Akascape/CTkTable/assets/127453231/e1227abb-ac14-4c04-a428-e46d1a398b79)

Is it not possible to make changes to a table with a value [[]]? Thanks in any help

**Comment by Akascape** [comment]
@tertris-me 
 You must need to define 1 row as header.

**Reply by tertris-me** [reply]
Thanks for the fast reply. How can I do that?

**Reply by AgniprabhaGanguly** [reply]
yeah
